core/domain/fs_domain.py

```diff
@@ -572,7 +304,7 @@ def open(self, filepath, version=None, mode='r'):
             mode: str. The mode with which to open the file.
 
         Returns:
-            FileStreamWithMetadata. The file stream domain object.
+            FileStream. The file stream domain object.
         """
         self._check_filepath(filepath)
         return self._impl.get(filepath, version=version, mode=mode)
```

Comment:
Do we still need version & mode, here and elsewhere?

Reply:
Removed.